pytorch_lightning/callbacks/progress/rich_progress.py

```diff
@@ -396,6 +400,10 @@ def on_validation_batch_end(self, trainer, pl_module, outputs, batch, batch_idx,
             self._update(self.val_progress_bar_id, self.val_batch_idx, self.total_val_batches)
         self.refresh()
 
+    def on_validation_end(self, trainer: "pl.Trainer", pl_module: "pl.LightningModule") -> None:
+        if trainer.state.fn == "fit":
```

Comment:
What's the problem if we remove this `if`?

Reply:
else during `.validate`, it will compute and display the val logs. For now, this is an issue with rich. Fixed it here: https://github.com/PyTorchLightning/pytorch-lightning/pull/11690